imod_coupler/drivers/dfm_metamod/dfm_metamod.py

```diff
@@ -59,6 +63,15 @@ def __init__(
         self.coupling = self.dfm_metamod_config.coupling[
             0
         ]  # Adapt as soon as we have multimodel support
+        self.dflow1d_lookup = dict[tuple[float, float], int]()
+        self.map_active_mod_dflow1d = dict[str, csr_matrix]()
+        self.mask_active_mod_dflow1d = dict[str, NDArray[np.int_]]()
+        mapping_input_dir = config_dir / "exchanges"
+        self.dflow1d_lookup, _ = get_dflow1d_lookup(mapping_input_dir)
+        (
+            self.map_active_mod_dflow1d,
+            self.mask_active_mod_dflow1d,
+        ) = mapping_active_mf_dflow1d(mapping_input_dir, self.dflow1d_lookup)
```

Comment:
added it to the top. I have created an issue IMOD6-963 for storing the paths of all the mapping files in the toml file. I will move the method call to initialize as part of that task, because now it is convenient to do it here